src/message/MessageListWeb.js

```diff
@@ -28,8 +28,9 @@ type Props = {
 };
 
 export default class MessageListWeb extends Component<Props> {
-  webview: ?Object;
+  webview: WebView;
```

Comment:
I don't think this is the correct type of the reference to the WebView.
Also, it has to be optional with a `?`
https://flow.org/en/docs/react/refs/